docs/application/tizen-studio/release-notes/6-0-release-notes.md

```diff
@@ -0,0 +1,83 @@
+  - Update .NET workload option is added.
+- Visual Studio (Windows)
+  - Added UnitTest and Code Coverage support for Tizen Native Applications.
+  - Added ASAN and LSAN tools support for Tizen Native Applications.
+  - Added right click menu named "Install Tizen Application" to install application directly on a target device.
+  - Added Issue Reporter menu to report issues related to VS plugin.
+  
+
+### Fixed bugs
+  - Fixed the issue of Tizen studio launcher splash image getting flipped in macOS Sonoma.
+  - Fixed the issue of rpk packages not installing in tv using sdb.
+  - Fixed the issue of .NET workload getting installed on every launch in VSCode plugin.
+  - Fixed the build issue of Tizen Native Shared Library projects in VSCode plugin.
+  - Fixed the issue of newly added source and resource files not getting updated in Native project config in VSCode plugin.
+  - Fixed the issue of tizen-manifest.xml not opening for native and TizenNUIGadget_inhouse project in VS plugin.
+  - Fixed the issue of all files are deleted when creating a project inside an existing solution in VS plugin.
+  - Fixed the issue of dotnet trace gets stuck in VS plugin.
+  - Fixed the issue of certificate change not getting impacted by prompting user to restart VS.
+  - Fixed the issue of application launching in unselected device in VS plugin.
+  - Fixed the issue of Dotnet profiler session not closing on creation of another project in VS plugin.
+
+### Known issues
+
+  - On macOS, if the UI perspectives are not displayed properly after updating Tizen Studio, it is recommended to restart the IDE. This issue is because of the uncleared cache from the previous installation.
+  - In case you see any old perspective, it is recommended to create a new workspace and import the projects to the new workspace to resolve the issue.
+  - On macOS, since Catalina and above versions, Native Templates-5.5 will not build with CLI when the compiler is set to **GCC**.
+- Common
+  - If you install Tizen Studio in a directory that requires administrator privileges for access, an alert will be given. For example, for **C:\ProgramFiles**, location administrator privilege is needed to run Tizen SDK tools. Tizen Installer and Baseline SDK Installer alerts you if you try to install it into such a directory.
+  - On macOS, Tizen Studio won't work properly if it is installed on Desktop directory, that is **$HOME/Desktop**. However it works properly if it is installed on any other directory.
+- Web and Native IDE
+  - You can create unit tests for Tizen 2.3.2 and higher version projects only. Unit testing for older versions is not supported.
+- Web IDE
+  - The preview tab in the Web Page Editor sometimes does not appear properly. In this case, you can use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl + 4** of the Web SDK HTML Editor.
```

Comment:
```suggestion
  - The preview tab in the Web Page Editor sometimes does not appear properly. In this case, you can use an alternative feature, named Web SDK HTML Editor, which has enhanced features compared to the Web Page Editor. Instead of the preview tab in the Web Page Editor, use the preview feature **Ctrl+4** of the Web SDK HTML Editor.
```